src/plugins/kibana_usage_collection/server/collectors/ops_stats/ops_stats_collector.ts

```diff
@@ -33,6 +33,11 @@ export function getOpsStatsCollector(
     // Ensure we only include the same data that Metricbeat collection would get
     // @ts-expect-error
     delete metrics.process.pid;
+    for (const process of metrics.processes) {
+      // @ts-expect-error
+      delete process.pid;
+    }
```

Comment:
Not directly related to the PR as this was done already, but I see we're `cloneDeep`'ing the metrics just to remove a few fields. Can't we avoid this costly call with just spreads and/or omits?

Reply:
i agree we can improve this logic. I'd suggest we do it in a different RP though